internal/pkg/manifest/manifest_test.go

```diff
@@ -472,6 +513,69 @@ func fullStruct() *Content {
 					},
 				},
 			},
+			{
```

Comment:
A tu je to iste co v JSON, len reprezentovane Go strukturami.
Testuje sa prevod:
- `JSON` -> `Go`, vid `TestManifestLoad`
- `Go` -> `JSON`, vid `TestManifestSave`